src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -284,9 +149,12 @@ define(function (require, exports, module) {
                     <div
                         ref="container"
                         className={containerClasses}
-                        onClick={this._handleContainerClick}
-                        onClickCapture={this._handleContainerClickCapture}>
-                        {childComponents}
+                        onClick={this._handleContainerClick}>
+                        <LayersGroup
+                            isRoot={true}
+                            disabled={this.props.disabled}
+                            document={doc}
+                            layerNodes={doc.layers.roots}/>
```

Comment:
Hmm, `doc.layers.roots` indeed is a list of `LayerNode` models. Prove: https://github.com/adobe-photoshop/spaces-design/blob/master/src/js/models/layernode.js#L95-L103